packages/broker/src/helpers/config.schema.json

```diff
@@ -1,304 +1,350 @@
 {
```

Comment:
Why these changes? Are these just indentation?

Reply:
Ah must have autoformatted the whitespace. It's now 4 space indentation like rest of code.